arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/TxSkipSection.java

```diff
@@ -124,27 +124,30 @@ public void resolvePostTransaction(Hub hub, WorldView state, Transaction tx, boo
     final AccountFragment senderAccountFragment =
         hub.factories()
             .accountFragment()
-            .make(
+            .makeWithTrm(
                 senderAccountSnapshotBefore,
                 senderAccountSnapshotAfter,
+                senderAddress,
                 DomSubStampsSubFragment.standardDomSubStamps(hub.stamp(), 0));
 
     // recipient account fragment
     final AccountFragment recipientAccountFragment =
         hub.factories()
             .accountFragment()
-            .make(
+            .makeWithTrm(
```

Comment:
This is required by the arithmetization, specifically: in the permuted domain the first appearance of an account should be with some trimming.